chesster.js

```diff
@@ -51,6 +80,95 @@ var channels = {
         return "<#"+this.getId(name)+">";
     }
 };
+var channelsByID = { };
+
+
+var fuzzy_match = function (search_string, targets) {
+    if (!targets) { return null; } // TODO: not sure this is the right choice ...
+    var results = [];
+    targets.forEach(function(item) {
+        var distance = levenshtein.get(search_string, item);
+        results.push([distance, item]);
+    });
+    results.sort(function(a,b) {
+        return a[0] - b[0];
+    });
+    var choices = [];
+    var min_distance = results[0][0];
+    results.forEach(function(item) {
+        var distance = item[0];
+        var match = item[1];
+        if (distance == min_distance) {
+            choices.push(match);
+        }
+    });
+
+    return choices;
+}
+var get_command_and_targets = function(message, commands, arg_string) {
```

Comment:
I dont mind defining functions this way - as far as I know, it is effectively the same, just a style difference - but I would like to be consistent.

Can we make this consistent with other function definition style? or make the others the same as this. Either way is fine with me.